hw/ip/uart/data/uart.hjson

```diff
@@ -61,10 +61,14 @@
       type:    "status",
       default: "1",
       desc:    "raised if the transmit FIFO is past the high-water mark."}
+    { name:   "tx_empty",
+      type:   "status",
+      default: "1",
```

Comment:
I'm intrigued by the 'default' value of an interrupt, please?
I can't see it used anywhere other than the uart IP block; is this something that other IPs should be setting appropriately, perhaps for Status type interrupts?

Reply:
I think it may be used by some of the autogen stuff for interrupt testing so it's aware this interrupt will be 1 out of reset rather than 0. Though if this is not being used elsewhere may be we've got another way to do this and I should just drop this line.

Reply:
Just checked it also employed on flash_ctrl at least

https://github.com/lowRISC/opentitan/blob/a182fcef27866fb502d8cceb810bdf76cfa3ef42/hw/ip_templates/flash_ctrl/data/flash_ctrl.hjson.tpl#L68-L77

It's used by autogen tests (e.g. so you know when you read from interrupt status after reset it will give a `1` for these interrupts).